src/nodes.coffee

```diff
@@ -1945,7 +1945,7 @@ exports.Code = class Code extends Base
       # encountered, add these other parameters to the list to be output in
       # the function definition.
       else
-        if param.isComplex()
+        if param.isComplex() or param.isCall()
```

Comment:
I just couldn’t remember what `isComplex` meant 😄 _But_ I did remember that we documented it [here](https://github.com/jashkenas/coffeescript/wiki/%5BHowTo%5D-How-parsing-works#nodescoffee), which says:
> If `node.isComplex() is false`, it is safe to use `node` more than once. Otherwise you need to store the value of `node` in a variable and output that variable several times instead. Kind of like this: `5` is not complex. `returnFive()` is complex. It’s all about “do we need to cache this expression?”

So there it is—`returnFive()` is exactly our use case. Thanks for the suggestion!